de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java

```diff
@@ -648,9 +648,7 @@ public void show(final WindowConfig config,
 
     @Override
     public void submitUserFeedback(Splittable splittable, final IsHideable isHideable) {
-        StringQuoter.create(userInfo.getUsername()).assign(splittable, "username");
-        StringQuoter.create(Window.Navigator.getUserAgent()).assign(splittable, "User-agent");
-        feedbackServiceProvider.get().submitFeedback(splittable, new AsyncCallback<Void>() {
+       userSupportServiceProvider.get().submitSupportRequest(splittable, new AsyncCallback<Void>() {
```

Comment:
If we're renaming the service call from `submitFeedback` to `submitSupportRequest`, should we rename this presenter method too? Also, what about the `DesktopView`: are we still letting users submit feedback from the desktop help menu?